src/otx/config/data/mmdet.yaml

```diff
@@ -0,0 +1,14 @@
+defaults:
+  - default
+
+data_format: coco_instances
+
+subsets:
+  train:
+    batch_size: 8
+    num_workers: 2
+    transform_lib_type: MMDET
```

Comment:
What does `transform_lib_type` mean? You mean like these transforms are coming from the mmdetection library?